### DISCUSSION TITLE
太空太陽能

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#726] [General]
中國一家叫Longi的公司開發了太空太陽能，預計比現有的任何能源都環保，目前的可再生能源大部分都無法可靠的發電，而太空太陽能是發射太陽能板到太空，沒有臭氧層過濾紫外線，也不會佔據地球上的土地，雖然聽起來很好，但卻難以實行，原因在於第一點，發射成本過高，即便到目前成本已降低許多，第二點是能量的傳輸效率低。目前Longi正與大學以及政府開發傳輸技術和低成本運載火箭，希望有朝一日能夠實行，解決地球的能源問題。

[https://medium.com/predict/the-ultimate-clean-energy-is-about-to-be-unlocked-c6fe66dfbd19](https://medium.com/predict/the-ultimate-clean-energy-is-about-to-be-unlocked-c6fe66dfbd19)

**Comment by Chiway1229** [comment]
那個 連結好像有問題進不去喔
我認為解決地球能源問題的確是重要的課題
不過太陽能板對地球資源的消耗也是非常大的 尤其製作太陽能的面板需要耗費許多稀土
感覺世界上對能源問題還沒有比太陽能更好的解決方案

**Comment by AnAnistired** [comment]
其實很久很久以前就有這種概念，只是電力的傳輸真的需要很大的突破，才有可能實際應用

**Comment by williamyen0623** [comment]
我相信科學家能精準的計算射上去台陽能版的軌道，但不可否認的是，會有未知的外在因素使得太陽能板墜回地球，那將是非常危險的

**Comment by manipulateed** [comment]
能為地球即將枯竭的能源做出貢獻與實際行動，真的要很謝謝他們的付出。

**Comment by danwan1027** [comment]
希望能快點找到永續能源，別再傷害地球了

**Comment by Yunhans** [comment]
我記得這個很多衛星都已經在使用了，問題是要如何把能源傳回地球

**Comment by WEI-CHEN-CHUN** [comment]
電力傳輸會需要非常強的電磁波，目前實行上仍有困難

**Comment by Pietra1226** [comment]
在太空主要的能源就是使用太陽能，只是要怎麼傳回地球是個問題，感覺會更耗能也可能更花錢。

**Comment by Evan0979** [comment]
太空太陽能感覺效率會超高欸

**Comment by CPCpyt** [comment]
我覺得從環境上的角度來說，
這樣的建構方式甚至能夠有效改善原本太陽能的短版，
如占空間和土地汙染等。
